src/sentry/relay/globalconfig.py

```diff
@@ -5,7 +5,13 @@
 from sentry.utils import metrics
 
 # List of options to include in the global config.
-RELAY_OPTIONS: List[str] = []
+RELAY_OPTIONS: List[str] = [
+    "profiling.profile_metrics.unsampled_profiles.allowed_org_ids",
+    "profiling.profile_metrics.unsampled_profiles.excluded_org_ids",
+    "profiling.profile_metrics.unsampled_profiles.excluded_project_ids",
```

Comment:
I think we should make a clear distinction here:

1. When the feature is enabled or disabled per org, use the [existing feature handler](https://github.com/getsentry/getsentry/pull/11335/files) (i.e. no need to propagate the option directly to Relay).
2. When the feature is enabled or disabled per project, use a project feature handler similar to the one linked above.
3. Only when a flag is independent of the org or project (i.e. overall sample rate, list of platforms), propagate the option directly to Relay.

```suggestion
```

Reply:
Hi @jjbayer,  a few follow up questions to your comment here since I'm familiar enough with `options` but not much with the `FeatureHandler` specifically:

Regarding **point 1**: without an option defined specifically, how would I add or remove quickly a org from this list?
I think the idea of options is to have a quick mechanism to immediately shutdown stuff.  
If I have to add or remove things in sentry or getsentry the deploy time CI/test/deployment would be too long as opposed to Options Automator that take less than a minute. 

My understanding is that `OptionBackedRolloutFeatureHandler` will automatically create and register the 3 following options: 
* `my-feature.la-rollout`
* `my-feature.ea-rollout`
* `my-feature.ga-rollout`

So those would be available for modification in Options Automator as well, but we won't have any options to specifically only include or exclude a single project/org quickly through automator. We'd have to modify "la_orgs" for that and wait for the whole CI/CD stuff. Please correct me if I'm wrong on this.


Regarding **point 2**: what would be an equivalent of AutoRegisterOptionBackedRolloutFeatureHandler  but for projects that has an equivalent of `la_orgs`  but for projects? let's say `la_projects` ?

Reply:
> I think the idea of options is to have a quick mechanism to immediately shutdown stuff.

That's a good point. So far, we don't want to put org / project IDs in global config because external Relays also get them. Let me check with the team whether there's a way around that.

> We'd have to modify "la_orgs" for that and wait for the whole CI/CD stuff. Please correct me if I'm wrong on this.

Correct, that's why for example the span extraction feature handler uses a list of project IDs loaded from options, not from a hard coded constant.

> what would be an equivalent of AutoRegisterOptionBackedRolloutFeatureHandler but for projects that has an equivalent of la_orgs but for projects? let's say la_projects ?

I don't think it exists. You can either create it, or do everything manually like we did [here](https://github.com/getsentry/getsentry/blob/48ff2f5f3464ec8afd8e1315c0dae9fe03645f47/getsentry/features.py#L1586-L1598).

Reply:
perfect, modified as we discussed over the call. Thanks @jjbayer 🥔